src/app/modules/navigator/test-exec-navigator.component.css

```diff
@@ -1,10 +1,116 @@
+.testRunButton {
+  outline: 0;
+  display: inline-flex;
+  align-items: center;
+  justify-content: space-between;
+  min-width: 150px;
+  max-width: 150px;
+  color: #FFFFFF;
+  font-size: 14px;
+  letter-spacing: 1.1px;
+  overflow: hidden;
+  cursor: pointer;
+  float: left;
+}
+
+.testRunDropdown .successful li:before {
+  position: absolute;
+  margin:3px 0px 0 -25px;
+  font-family: FontAwesome;
+  content: "\f058"; /* check */
+  color: #00cc00;
+}
+
+.testRunDropdown .failure li:before {
+  position: absolute;
+  margin:3px 0px 0 -25px;
+  font-family: FontAwesome;
+  content: "\f071"; /* warning */
+  color: #dd0000;
+}
+
+.testRunDropdown .running li:before {
+  position: absolute;
+  margin:3px 0px 0 -25px;
+  font-family: FontAwesome;
+  content: "\f059"; /* question mark */
+  color: #808080;
+}
+
+.testRunDropdown .failed li:before {
+  position: absolute;
+  font-family: 'FontAwesome';
+  content: '\f071';
+  margin:3px 0px 0 -25px;
+  color: #f00;
+}
+
+.testRunButton span {
+  padding-left: 10px;
+}
+
+.testRunButton:focus + .testRunDropdown,
+.testRunButton:active + .testRunDropdown,
+.testRunDropdown:active {
+  -webkit-transform: translate(20px, 30px);
+  transform: translate(20px, 30px);
+  opacity: 1;
+  visibility: visible;
+}
+
+.testRunButton + .testRunDropdown {
+  position: absolute;
+  background: #202020;
+  width: 100%;
+  border-radius: 4px;
+  box-shadow: 0 4px 12px rgba(0, 0, 0, 0.1);
+  text-align: left;
+  opacity: 0;
+  visibility: hidden;
+  transition: 0.3s ease;
+  max-height: 100px;
+  max-width: 350px;
+  overflow-y: auto;
+  overflow-x: hidden;
+}
+
+.testRunButton + .testRunDropdown li {
+  z-index: 2;
+  position: b;
```

Comment:
found only "apocryphal" hints of this being a valid value for the position attribute. Does this do anything? Is this standard css?